templates/project/sdkconfig.defaults

```diff
@@ -57,3 +57,6 @@ CONFIG_ESP_COREDUMP_CHECKSUM_SHA256=y
 CONFIG_ESP_COREDUMP_ENABLE=y
 CONFIG_ESP_COREDUMP_MAX_TASKS_NUM=64
 CONFIG_ESP_COREDUMP_DECODE="disable"
+
+# NTP Config
+CONFIG_LWIP_SNTP_UPDATE_DELAY=86400000
```

Comment:
can save this for a template update PR if preferrded

Reply:
Same thoughts. I'm not sure why we should override the default of an hour.